test/unit/spec/appWidgets/input/selectInputSpec.js

```diff
@@ -64,10 +64,10 @@ define(['common/runtime', 'widgets/appWidgets2/input/selectInput'], (Runtime, Se
             });
         });
 
-        it('Should start and stop a widget', (done) => {
+        it('Should start and stop a widget', () => {
```

Comment:
The content of most tests is the same, I updated to return Promises that get resolved instead of using `done`. IME it's a little more stable, though I don't recall many problems with this spec.